pkg/workloads/lib/package.py

```diff
@@ -94,6 +123,10 @@ def build(args):
     try:
         build_packages(python_packages, ctx.bucket)
         util.log_job_finished(ctx.workload_id)
+    except CortexException as e:
+        e.wrap("error")
+        logger.exception(e)
+        ctx.upload_resource_status_failed(*python_packages_list)
     except Exception as e:
         logger.exception(e)
```

Comment:
is there a reason to not `e.wrap("error")` in this case and consolidate the error handling? Also do the other workload top-level python files need a similar change?